.github/workflows/build_test_publish_images.yaml

```
@@ -83,12 +83,34 @@ jobs:
 
       - name: Compute cuopt version
         id: compute-cuopt-ver
+        env:
+          NIGHTLY_INDEX: https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:26:32: Double quote to prevent globbing and word splitting

(shellcheck)